src/clusterfuzz/_internal/issue_management/issue_filer.py

```diff
@@ -455,6 +455,11 @@ def file_issue(testcase,
 
   for cc in ccs:
     issue.ccs.add(cc)
+  # Add labels from crash metadata.
+  for crash_subtype in testcase.crash_subtypes:
```

Comment:
Could we place this closer to the existing crash_type handling on line 370?

Reply:
Done.